modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -6,15 +6,15 @@
 include::partial$configuration/premium-upgrade-promotion-option.adoc[]
```

Comment:
```suggestion
include::partial$configuration/premium-upgrade-promotion-option.adoc[leveloffset=+1]
```
Add this and only indent the `Example` heading in the option partial.

Reply:
@MitchCTiny that wouldn't work... the current heading level in that file is correct: https://github.com/tinymce/tinymce-docs/blame/hotfix/6.2/DOC-1793/modules/ROOT/partials/configuration/premium-upgrade-promotion-option.adoc#L1

Reply:
Honestly though, I don't think that heading should be in the option. It should be on this page instead 🤷

Reply:
Reworked things a little as discussed on our slack call.